<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor::DisplaySettingsPage</class>
 <widget class="QWidget" name="TextEditor::DisplaySettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxDisplaySettings">
     <property name="title">
      <string>Display Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="showWrapColumn">
          <property name="text">
           <string>Display right &amp;margin at column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wrapColumn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableTextWrapping">
        <property name="text">
         <string>Enable text &amp;wrapping</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="displayLineNumbers">
        <property name="text">
         <string>Display line &amp;numbers</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="visualizeWhitespace">
        <property name="text">
         <string>&amp;Visualize whitespace</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="displayFoldingMarkers">
        <property name="text">
         <string>Display &amp;folding markers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="highlightCurrentLine">
        <property name="text">
         <string>Highlight current &amp;line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showWrapColumn</sender>
   <signal>toggled(bool)</signal>
   <receiver>wrapColumn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
